platforms/bukkit/src/main/java/com/pg85/otg/bukkit/commands/SpawnCommand.java

```diff
@@ -1,25 +1,25 @@
 package com.pg85.otg.bukkit.commands;
```

Comment:
Awesome, ofc you realise that the first thing ppl will want is an undo function and a holo-BO3 they can move around before placing :P.